plugins/chat/spec/lib/chat_sdk/channel_spec.rb

```diff
@@ -38,4 +38,90 @@
       end
     end
   end
+
```

Comment:
I decided to implement start_reply/stop_reply in channel and not in both channel and thread for simplicity